### PR TITLE
fix unsupported token panic

### LINKED ISSUE #58: [bug] Commenting out an iso literal should not cause the compiler to panic
* Maybe we can handle this by adding `/` or `//` to the lexer and then just bailing if we encounter it

### PR DESCRIPTION
**Pull request by ch1ffa**
This PR adds a new error token type to handle unexpected single line comment token `//` and collects all other encountered unexpected tokens.

Closes #58

### CODE REVIEW

**Comment by rbalicki2** [comment]
I'm still seeing panics when I comment out a line. Maybe something broke between when you tested and pushed or we're missing some changes?

File: crates/isograph_lang_parser/src/token_kind.rs [hunk 1]
```diff
@@ -75,6 +75,9 @@ pub enum IsographLangTokenKind {
     // Comments
     // #[regex("#[^\n\r]*")]
     // SingleLineComment,
+    #[regex("//")]
```
Comment (rbalicki2) [inline crates/isograph_lang_parser/src/token_kind.rs:78]: you should be able to use `#[token("//")]`, which presumably is faster than using a regex. Source: https://github.com/maciejhirsz/logos

Comment (ch1ffa) [reply crates/isograph_lang_parser/src/token_kind.rs:78]: Ah, my bad, thanks! Was testing which is better: show a token error of the whole "commented" line

Comment (ch1ffa) [reply crates/isograph_lang_parser/src/token_kind.rs]: @rbalicki2 done!

**Comment by rbalicki2** [comment]
closed in favor of 9c01b96f6570a8a5599b65240cc1a18464c8110e and d60ff77a6eb5eba2a5db456d642cbc4e2f524831

**Pull request closed without merging**